pyglmnet/pyglmnet.py

```diff
@@ -446,6 +446,8 @@ class GLM(BaseEstimator):
     random_state : int
         seed of the random number generator used to initialize the solution.
         default: 0
+    callbacks: boolean
```

Comment:
why not make this a `callable` (instead of bool) which can be any function and is called at every iteration. If it's `None` nothing happens. So, you embed all the logic inside this callback

Reply:
a bit like this: https://github.com/alphacsc/alphacsc/blob/a87b48cd7229af4c5cee4f94a334e2790d21e750/alphacsc/learn_d_z.py#L84

Reply:
nice suggestion, done!